ethereum/contracts.go

```diff
@@ -22,4 +22,50 @@ type BridgeContract interface {
 		recipient []common.Address,
 		token []common.Address,
 	) (AssetsLockedIterator, error)
+
+	PastAssetsUnlockConfirmedEvents(
+		startBlock uint64,
+		endBlock *uint64,
+		unlockSequenceNumberFilter []*big.Int,
+		recipientFilter [][]byte,
+		tokenFilter []common.Address,
+	) ([]*MezoBridgeAssetsUnlockConfirmed, error)
+
+	ConfirmedUnlocks(arg0 *big.Int) (bool, error)
+}
+
+type MezoBridgeAssetsUnlockConfirmed struct { // TODO: Remove once bindings for `MezoBridge` are re-generated and contain features related to attestation.
```

Comment:
I think the `Confirmed` suffix should not be part of the struct name. This is just a bag of data, the context is added one level-above.
```suggestion
type MezoBridgeAssetsUnlocked struct { // TODO: Remove once bindings for `MezoBridge` are re-generated and contain features related to attestation.
```

Reply:
It's the same situation as in https://github.com/mezo-org/mezod/pull/506#discussion_r2261139091.
We shouldn't change anything until bindings for MezoBridgeV2 are available.
Once that happen, we will simply remove this method.